app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -203,6 +216,18 @@ class CasesService @Inject()(appConfig: AppConfig,
     )
   }
 
+  private def addReviewStatusChangeEvent(original: Case,
```

Comment:
can be formatted better
see the other methods

Reply:
Formatting is a personal preference based thing, I don't see any issue with how it is currently formatted?

Reply:
I've changed it for consistency